roles/kube_router/tasks/install.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Kube-router | oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:kube-router-node
+  oc_adm_policy_user:
+    user: system:serviceaccount:kube-system:kube-router
+    resource_kind: scc
+    resource_name: privileged
+    state: present
+
+- name: Kube-router | Set default selector for kube-system
+  run_once: true
+  command: >
+    {{ openshift_client_binary }}
+    --config={{ openshift.common.config_base }}/master/admin.kubeconfig
+    annotate  ns kube-system openshift.io/node-selector="" --overwrite
```

Comment:
That would remove master nodeselector for api/controller/etcd too. Maybe its worth moving kube-router to a separate NS instead?

Reply:
I have updated the code to create a new namespace. ¿It is ok now?